### PR TITLE
Navbar round2

### PR DESCRIPTION
**Pull request by darrenchay**
BIG CHUNGUS

### CODE REVIEW

File: client/src/components/utils/Footer/Footer.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Anchor, Footer as SiteFooter, Text } from "grommet";
 export default function Footer() {
   return (
     <SiteFooter
-      style={{ padding: 20, position: "fixed", bottom: 0, width: "100vw" }}
+      style={{ padding: 20, position: "static", bottom: 0, width: "100vw" }}
```
Comment (mbasak99) [inline client/src/components/utils/Footer/Footer.js:7]: This should probably be reverted back to "fixed" cuz 
image

File: client/src/components/utils/Navbar/Navbar.js [hunk 2]
```diff
@@ -148,7 +184,45 @@ export default function Navbar() {
...
+
+    loginOptions = 
+      <div>
+        <Button startIcon={<AccountCircleIcon className={classes.shoppingCartIcon}/>} className={classes.navButtons} onClick={redirectToLogin}>
+          Login
+        </Button>
+      </div>
+  }
+
+  console.log(location);
```
Comment (mbasak99) [inline client/src/components/utils/Navbar/Navbar.js:223]: Probably don't need this anymore.

**Review by mbasak99 — CHANGES_REQUESTED** [review]
Can you fix the places I commented on? Other than that it looks good.

File: client/src/Router.js [hunk 3]
```diff
@@ -9,10 +9,20 @@ import Navbar from "./components/utils/Navbar/Navbar";
...
+  let navbar;
+
+  if (window.location.pathname.includes("checkout")) {
+    console.log(window.location.pathname);
+    navbar = <div></div>
+  } else {
+    console.log(window.location.pathname);
+    navbar = <div><Navbar /></div>;
+  }
+
```
[the remark this answers is not in the document]
Comment (SachJay) [reply client/src/Router.js:21]: yah i dont feel like trying to fix it for what it adds

**Review by mbasak99 — APPROVED** [review]

**Review by SachJay — APPROVED** [review]

**Pull request merged**